Enable downloading Czech model

skvrnami opened the pull request:
I modified the `nametagger_download_model` function so that it can download both English and Czech NER models available from ÚFAL.

jwijffels commented:
Thanks for the proposed changes. Some remarks.

- Can you make sure the example with english / english-conll-140408 / czech-cnec-140304 work?
- Can you remove the use of paste0 and replace it with an appropriate paste command (there is no need to depend on a more later version of R)
- Can you remove the changes you did to src/RcppExports.cpp (there is no need to depend on a more later version of Rcpp)

skvrnami commented:
The last commit should put the code in line with your suggestions - I removed the changes to src/RcppExports.cpp and replaced `paste0` with `paste`.

I checked that the example works by running this code:
```
library(nametagger)

x <- data.frame(doc_id      = c(1, 1, 2),
                sentence_id = c(1, 2, 1),
                text        = c("I\nlive\nin\nNew\nYork\nand\nI\nwork\nfor\nApple\nInc.", 
                                "Why\ndon't\nyou\ncome\nvisit\nme", 
                                "Good\nnews\nfrom\nAmazon\nas\nJohn\nworks\nthere\n."))

en_m1 <- nametagger_download_model("english-conll-140408")
predict(en_m1, x)                          

en_m2 <- nametagger_download_model("english")
predict(en_m2, x)

x_cz <- data.frame(doc_id = 1, 
                   sentence_id = 1, 
                   text = "Policajti\nkradou\nkočkám\nmléko.")

cz_m1 <- nametagger_download_model("czech-cnec-140304")
predict(cz_m1, x_cz)

```

It worked fine.

jwijffels commented:
I've made the changes to your repo which I think we required. 
Looks ok now, I'll incorporate it. Thanks for the changes.
